File: packages/js/core/src/manifest/formats/web3api.infra/migrators/0.0.1-prealpha.1_to_0.0.1-prealpha.2.ts
/* eslint-disable @typescript-eslint/naming-convention */

import { InfraManifest as OldManifest } from "../0.0.1-prealpha.1";
import { InfraManifest as NewManifest } from "../0.0.1-prealpha.2";

export function migrate(old: OldManifest): NewManifest {
  return {
    ...old,
    format: "0.0.1-prealpha.2",
  };
}

File: packages/py/manifests/polywrap/src/formats/polywrap.infra/migrators/_0_0_1_prealpha_1_to__0_0_1_prealpha_2.py
from ... import import_manifest
from typing import TypeAlias
import os

local_path: os.PathLike = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))

__0_0_1_prealpha_1 = import_manifest('0.0.1-prealpha.1', local_path)
OldManifest: TypeAlias = __0_0_1_prealpha_1.InfraManifest
__0_0_1_prealpha_2 = import_manifest('0.0.1-prealpha.2', local_path)
NewManifest: TypeAlias = __0_0_1_prealpha_2.InfraManifest

def migrate(old: OldManifest) -> NewManifest:
    new: NewManifest = old
    new.format = "0.0.1-prealpha.2"
    return new